code/datums/shuttles.dm

```diff
@@ -227,6 +229,10 @@
 			description = params["new_template_description"]
 			update_static_data(usr, ui)
 			return TRUE
+		if("setTemplateTags")
```

Comment:
same here -- you'd need to make a button that fires off an act() containing, as a list, the new tags for the template

Reply:
Same, it's fully supported now, and this has been split into addTemplateTags and removeTemplateTags.